### PR TITLE
Fixing bugs with ionic 4 breakpoints shows unverified

### PR DESCRIPTION
**Pull request by SounD120**
Fix for #426
Updated Chrome debug core version

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -538,8 +680,8 @@
     "source-map-support": "^0.4.18",
     "temp": "^0.8.3",
     "vscode": "^1.1.6",
-    "vscode-chrome-debug-core": "3.21.2",
-    "vscode-debugadapter": "^1.22.0",
+    "vscode-chrome-debug-core": "^6.7.19-patch.0",
```
Comment (dhanvikapila) [inline package.json]: Should we pick up the latest version `6.7.23`

File: package.json [hunk 2]
```diff
@@ -107,8 +107,17 @@
             "target": "device",
             "port": 9222,
             "sourceMaps": true,
-            "cwd": "${workspaceRoot}",
-            "ionicLiveReload": false
+            "cwd": "${workspaceFolder}",
+            "ionicLiveReload": false,
+            "sourceMapPathOverrides": {
```
Comment (ruslan-bikkinin) [inline package.json]: Can we override sourcemap paths implicitly inside debugging session for ionic applications without forcing users to change their configs?

File: src/debugger/cordovaPathTransformer.ts [hunk 3]
```diff
@@ -34,98 +25,98 @@ export class CordovaPathTransformer extends BasePathTransformer {
...
-                resolve();
-                return;
-            }
+    public setBreakpoints(args: ISetBreakpointsArgs): ISetBreakpointsArgs {
+        if (!args.source.path) {
+            // sourceReference script, nothing to do
+            return args;
+        }
 
-            const url = utils.canonicalizeUrl(args.source.path);
```
Comment (ruslan-bikkinin) [inline src/debugger/cordovaPathTransformer.ts]: Why this block is removed? Is it now inside BasePathTransformer?

File: src/debugger/cordovaIosDeviceLauncher.ts [hunk 4]
```diff
@@ -229,7 +229,7 @@ export class CordovaIosDeviceLauncher {
     private static mountDeveloperImage(): Q.Promise<any> {
         return CordovaIosDeviceLauncher.getDiskImage()
             .then(function (path: string): Q.Promise<any> {
-            let imagemounter: child_process.ChildProcess = child_process.spawn("ideviceimagemounter", [path]);
+            let imagemounter: child_process.ChildProcess = child_process.spawn("ideviceimagemounter", [path, path + ".signature"]);
```
[the remark this answers is not in the document]
Comment (SounD120) [reply src/debugger/cordovaIosDeviceLauncher.ts]: I looked [this docs](http://docs.quamotion.mobi/en/latest/imobiledevice/ideviceimagemounter.html) and found out that the signature file stores like this
image
so i added this argument

File: src/debugger/cordovaPathTransformer.ts (hunk 3 above)
Comment (SounD120) [reply src/debugger/cordovaPathTransformer.ts]: I replaced all code from old path transformer by [Url path transformer](https://github.com/Microsoft/vscode-chrome-debug-core/blob/master/src/transformers/urlPathTransformer.ts) and adapted it to cordova tools

**Pull request merged**